tools/accuracy_checker/accuracy_checker/annotation_converters/oar3d.py

```diff
@@ -0,0 +1,121 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+from pathlib import Path
+import numpy as np
+
+from ..representation import OAR3DTilingSegmentationAnnotation
+from ..config import NumberField, StringField, PathField
+from .format_converter import DirectoryBasedAnnotationConverter
+from .format_converter import ConverterReturn
+
+
+class OAR3DTilingConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'ge_tiling'
+    annotation_types = (OAR3DTilingSegmentationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            "depth": NumberField(optional=True, default=32, description="Tile depth."),
+            "height": NumberField(optional=True, default=128, description="Tile height."),
+            "width": NumberField(optional=True, default=128, description="Tile width."),
+            "input": StringField(optional=True, default="inputs", description="Name of input data variable."),
+            "output": StringField(optional=True, default="outputs", description="Name of output data variable."),
+            "preprocessed_dir": PathField(optional=False, is_directory=True, check_exists=True,
+                                          description="Preprocessed dataset location")
+        })
+
+        return parameters
+
+    def configure(self):
+        self.data_dir = self.get_value_from_config('data_dir')
+        self.wD = self.get_value_from_config('depth')
+        self.wW = self.get_value_from_config('width')
+        self.wH = self.get_value_from_config('height')
+        self.input = self.get_value_from_config('input')
+        self.output = self.get_value_from_config('output')
+        self.preprocessed_dir = self.get_value_from_config('preprocessed_dir')
+
+    def convert(self, check_content=False, **kwargs):
+
+        data_folder = Path(self.data_dir)
+        preprocessed_folder = Path(self.preprocessed_dir)
+        input_folder = preprocessed_folder / 'input'
+        mask_folder = preprocessed_folder / 'mask'
+
+        for folder in [input_folder, mask_folder]:
+            if not os.path.exists(folder):
+                os.mkdir(folder)
```

Comment:
please use pathilb instead, we avoid to use several path processing libs in the same time